Personal portfolio site with title and header sections

Koshao-sys opened the pull request:
In this pull request i have worked on items below;
1. Created an html page with a title and header section
2. Added css for the title and header section
3. Documented my Readme file to share what the project is about and future features i will be adding.

Gambit142 commented on index.html:
```diff
@@ -0,0 +1,34 @@
+<body>
+    <header>
+        <div class="container-toolbar">
+            <h1 class="toolbar-title">Welcome</h1>
```
- [ ] Kindly make the logo a link. That is, you should use the `a` tag instead of the h1 tag. As a matter of convention and HTML best practices, logos are `a` tags. It is also semantic and provides users with great user experiences during the navigation of the webpage. Kindly implement this change. 👍

Gambit142 commented on index.html:
```diff
@@ -0,0 +1,34 @@
+        <div class="container-toolbar">
+            <h1 class="toolbar-title">Welcome</h1>
+            <span class="toggle"><img src="./Images//enabled.png" alt="nav"></span>
+        </div>
```
- [ ] Kindly change this from the `span` tag to the `nav` tag. Using the `nav` tag for navigation links and hamburger buttons is more semantic. 👍

Gambit142 commented on index.html:
```diff
@@ -0,0 +1,34 @@
+
+    <section class="headline">
+        <div class="headline-name-job">
+            <h2 class="headline-name">Hey there, I'm Koshao</h2>
```
- [ ] Kindly change this from the `h2` tag to the `h1` tag. This is the main header of the webpage and main headers should be embedded in `h1` tags for semantic purposes. 👍

Gambit142 commented on style.css:
```diff
@@ -0,0 +1,93 @@
+  top: 314px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  background-image: url("./Images//mobile-bg.svg");
+  background-repeat: no-repeat;
+  height: 648px;
+  width: 100%;
+}
+
```
- [ ] Kindly adjust the position of the background image. The header with text _**"Hey There, I'm Koshao"**_ is close to the log in the background image. In the Figma template, it is a little beneath it. Kindly adjust this and make the designs identical. 👍 

### Your Design

![Screenshot_log](https://user-images.githubusercontent.com/77106962/212133400-a961b6b8-a4b9-4c5e-ab80-8ff2fba90481.png)

### Figma Design

![Screenshot_log2](https://user-images.githubusercontent.com/77106962/212133677-f3f8f72d-1d06-47d6-9093-ccc0bf77991c.png)

Gambit142 reviewed (changes requested):
Hi @Koshao-sys 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Professional README.md file ✔️ 
- Pull request title is self-explanatory ✔️ 
- Pull request summary is descriptive ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Koshao-sys commented:
I have made the changes as advised below;
1. make the logo a link, using the 'a' tag instead of the h1 tag
2. changed the span tag to the nav tag and used hamburger buttons.
3. Changed the main headers from h2 tags to h1 tags.
4. I adjusted the position of the background image to look like what is in the template.

Ibizugbe commented on README.md:
````diff
@@ -1 +1,207 @@
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
````
[optional] I think you can get rid of this section since we haven't written any tests yet. having incomplete statements or sections not properly filled is not a good display of professionalism. 😉

Ibizugbe commented on README.md:
```diff
@@ -1 +1,207 @@
+
+### Deployment
+
+You can deploy this project using: Figma
```
[OPTIIONAL] I think you can get rid of this too if you don't have the right content yet. You can't deploy a webpage using figma😉

Ibizugbe commented on index.html:
```diff
@@ -0,0 +1,52 @@
+
+    <section class="headline">
+        <div class="headline-name-job">
+            <h1 class="headline-name">Hey there. I'm Koshao</h1>
+            <h1 class="headline-job">I'm a software developer</h1>
```
[OPTIONAL] I think it is not semantically appropriate to be having two main headings😉. I suggest you change `<h1 class="headline-job">I'm a software developer</h1>` to <span class="headline-job">I'm a software developer</span> and put it inside `<h1 class="headline-name">Hey there. I'm Koshao</h1>` for styling. you can then use a `<br>` tag between them so it starts on a new line.😉

Ibizugbe commented on style.css:
```diff
@@ -0,0 +1,105 @@
+  background-color: #e5e5e5;
+}
+
+.headline-name {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  padding-top: 30px;
+  padding-bottom: 0;
+  margin-bottom: 0;
+}
+
+.headline-job {
+  font-family: 'inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  color: #36b37e;
+  line-height: 52px;
+  margin: 0;
+  padding-top: 0;
+}
```
[Optional[I think Your intro header is looking a bit different from the original design in `font-weight`🤔

your webpage
<img width="268" alt="yourHey" src="https://user-images.githubusercontent.com/64643995/212259227-3a5fbb12-93e2-4c39-94ab-1742163bab31.png">

the design
<img width="243" alt="figmaHey" src="https://user-images.githubusercontent.com/64643995/212259340-f5563550-1110-40ad-8f61-d98ab84fbd1b.png">

it is a compulsory requirement that you stick to the design, I think the previous reviewer didn't take note of it.
kindly make the adjustments where appropriate😉

Ibizugbe reviewed (approved):
Hi @Koshao-sys 👋, 

![BigTvGIF (2)](https://user-images.githubusercontent.com/64643995/212259855-1ffa5e19-0a33-4a05-8e01-a29edacf095b.gif)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅

- [ ] Linters are passing.
- [ ] Pull request has a descriptive header and summart.

## Optional suggestions
_Please kindly check the comments below and try to make the changes🤗_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
